reference/multipart.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: page
+title: Multipart Encoding
+---
+
+Praxis has built-in support for handling "multipart/form-data" (link to RFC?) encoded requests and responses, in the form of the `Praxis::Types::MultipartArray` type, and the `Praxis::Responses::MultipartOk` response.
+
+
+## Praxis::Types::MultipartArray
+
+The `MultipartArray` type is an `Attributor::Collection` of `Praxis::MultipartPart` members that allows you to specify the handling of each part of a multipart request or response body.
```

Comment:
Slight wording change:
...that allows you to specify how to handle each of the existing parts of a multipart request or response body.